### DISCUSSION TITLE
I don't know what is wrong

### DISCUSSION THREAD

**Discussion by SiKi02** [smartcontractkit/full-blockchain-solidity-course-py#943] [Q&A] [answered]
Hi people, hope you are doing well.
I have an issue with the vs code terminal and I don't know how to fix it. When I write python version on terminal it says 2.7.16 and when I write python3 version its completely different. Also I tried to ignore this problem but the terminal won't work because Pip is not installed as well eventhough the download of python includes it on Mac. 
<img width="1440" alt="Captura de Pantalla 2022-02-03 a la(s) 9 17 14 p  m" src="https://user-images.githubusercontent.com/96997244/152466417-b44af1b9-bedd-4fc7-a5c3-51d7cf69fc65.png">
#

**Comment by cromewar** [comment, marked as answer]
Hello @SiKi02 
Well this is not an error, your MacOS have two python versions 2.7 and 3.x which is normal, so depending on the one you want to use you should use python3 or python. To install pip3 follow [this](https://www.delftstack.com/howto/python/python-install-pip3-mac/) guide, and once you have installed it you should install the packages like this:

`pip3 install eth-brownie` 

I hope this info might help.

**Reply by SiKi02** [reply]
Thank you so much bro! @cromewar

**Reply by cromewar** [reply]
You are welcome
